accessibility/alt-text.md

```diff
@@ -47,6 +49,26 @@ No. If an image has no meaning, or if it's providing information that's redundan
+
+### How do screen readers render alt text? 
+
+Screen readers look at all of the HTML markup on a page, then announce the contents to the user in different ways depending on what kind of element it's describing. For example a link might be announced as something like "Link: my cool website address". A navigation menu might be announced like "Navigation: breadcrumb". These are examples of HTML element nodes. Screen reader users can command their screen reader to move around the page and describe different things at will. They can move forwards or backwards through text, read text at the character level (for example to confirm spelling), and pause and restart later, among other useful operations. 
+
+Many HTML element nodes can contain attributes. [A link could (but shouldn't) use a `title` attribute](https://github.com/springernature/frontend-playbook/blob/main/accessibility/common-remediations/html-attributes.md)), like this: `<a href="/" title="don't do this">my cool website address</a>`. The alt attribute is also of this type: `<img alt="my cool alt text" src="...">`. Attributes can't be interacted with in the same way as the rest of the page - you can't move forwards or backwards through the text, you can't read the text at the character level, and you can't pause the screen reader and restart later where you left off. 
+
+Imagine you're sitting at a bus stop, and the bus company has added a digital marquee to the shelter to give you important information. You can't control the marquee at all, you can only look at it, passively, on its own schedule. If you're momentarily distracted, causing you to miss the information you needed about your bus, the only thing you can do is wait for it to return to the start and then (eventually) get to the part you were interested in. Tiresome, right? This is similar to the experience of using alt text with a screen reader. Although the screen reader user can go back to start whenever they want, if your alt text is 1000 characters long and the information they need to hear again starts after 850 words, this person is going to be waiting for a very long time. 
+
+If the information is complex (and much of Springer Nature's scientific output is highly complex), the cognitive load on the screen reader user is higher, as they have more information to process and recall. For many (but not all) sighted users, it's a simple task to scan your eyes back and forth to read a complex sentence or sentence fragment again. It's impossible for a blind screen reader to do this with alt attributes, and this is the reason that alt text must be succinct. 
+
+Don't make it _so_ succinct that the information in the image is lost, but _do_ think about the cognitive burden of alt text as a whole; and consider if it's really alt text in the HTML that you need, or if your users would be better served with a full text transcript that they can directly interact with instead. 
```

Comment:
Given that transcripts are most commonly associated with audio/video, would it be better to instead reference something like a caption?

Reply:
hm. it's an interesting question. _We_ often leverage the caption for this on our journal content, but it's not going to be appropriate for every use case - when I wrote this, I was mainly thinking of [full text transcripts like this one from the CDC](https://www.cdc.gov/ncbddd/disabilityandhealth/infographic-disability-impacts-all.html) (scroll down to the bottom of the page to see it), which isn't in a caption and doesn't really need to be. Also thinking of stuff like transcripts that live under progressive disclosure widgets (could use `details/summary`, could just use a load of divs with an aria-suguared button) that are used on a lot of our immersive articles. 

I'm hesitant to specify a particular element here, I guess, as it's unnecessarily prescriptive. What I do want to do is add another page to this guide that just covers different options for text transcripts (and then I'll link to it here). Does that assuage your concerns? 🤞